### PR TITLE
Documentation: container profiles

### PR DESCRIPTION
**Pull request by alban**

### CODE REVIEW

File: Documentation/profiles.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+- Namespaces
+  - cgroup
+  - ipc
+  - mnt
+  - net
+  - pid
+  - user
+  - uts
+
+TODO
```
Comment (surajssd) [inline Documentation/profiles.md]: do you plan to add these todos over the course of time later or in the same PR ?

Comment (alban) [reply Documentation/profiles.md]: I was thinking over the course of time later.

File: Documentation/profiles.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  (e.g. `devices` cgroup subsystem)
+- *seccomp:* some system calls are restricted in the container
+- *Linux Security Modules (LSMs):* AppArmor, SELinux
+- *Volumes:* e.g. a host volume
+
+Offering a test environment for all possible configuration would be a
+combinational explosion. Instead, the Container Escape Bounty project defines a
+few _container profiles_ with predefined configurations. Security researchers
+can then start containers under one container profile. The bounty reward could
+be different depending on which container profiles the vulnerability is
```
[the remark this answers is not in the document]
Comment (alban) [reply Documentation/profiles.md]: Maybe the vulnerability is exploitable on a few container profiles. Not sure whether `profiles` should be plural or not.

File: Documentation/profiles.md (hunk 1 above)
Comment (surajssd) [reply Documentation/profiles.md]: cool LGTM

File: Documentation/profiles.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (indradhanush) [reply Documentation/profiles.md]: If we use `profiles` then the `vulnerabity` would need to become plural as well. In that case the sentence would become:
```...depending on which container profiles the vulnerabilities are exploitable on.```

**Pull request merged**